examples/bb84.py

```diff
@@ -125,7 +125,9 @@ def main(num_qubits=8):
     # Run simulations.
     repetitions = 1
 
-    result = cirq.Simulator().run(program=circuit, repetitions=repetitions)
+    result = cirq.Simulator(split_untangled_states=False).run(
```

Comment:
Why split=False?

Reply:
Seems just to be the RNG seed. (This is not guaranteed to succeed, the seed was specified here as an example where it does. We can choose another one.)